src/utils/ViewMapLeaf.ts

```diff
@@ -0,0 +1,98 @@
+class ViewMapLeaf<T> {
+  public containerType?: ContainerType;
+  public parent: ViewMapLeaf<T>;
+  public childs: ViewMapLeaf<T>[];
```

Comment:
I don't insist on changing it, but I've never seen such word before.

http://docs.oracle.com/javase/7/docs/api/javax/swing/tree/TreeNode.html#children()
https://api.jquery.com/children/